src/NzbDrone.Core/Notifications/Webhook/WebhookSeries.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.Notifications.Webhook
+{
+    internal class WebhookSeries
+    {
+
+        public string Id { get; set; }
```

Comment:
Nope, nice catch, was probably because I was autogenerating them and it still had the .ToString().

I'll update